cli/command/formatter/displayutils_test.go

```diff
@@ -0,0 +1,88 @@
+package formatter
+
+import "testing"
+
+func TestEllipsis(t *testing.T) {
+	str := "t🐳ststring"
+	newstr := Ellipsis(str, 0)
+	if newstr != "" {
+		t.Fatalf("Expected null, got %s", newstr)
+	}
```

Comment:
These tests would be great as a test table.

```go
var testcases = []struct{
    source string
    width int
    expected string
} {
    {source: "t🐳ststring", width: 0},
    ...
}

for _, testcase := range testcases {
    assert.Equal(t, testcase.expected, Ellipsis(testcase.source, testcase.width))
}
```

Also please use `assert.Equal()` instead of `if x != y { t.Fatalf(...) }`

Reply:
Modified. Sorry for the delay.

Reply:
Although the pr is merged, if u are free @dnephin  , I have a low level quetion why should we use `assert.Equal()` instead of `if x != y { t.Fatalf(...) }`? What's the disadvantages of `t.Fatalf`? (☉㊀☉)

Reply:
@charrywanganthony 

* `assert.Equal()` can't lie. It will always print the real value for the two values being compared. If you write `if x != y { t.FatalF(msg) }` the `msg` could be incomplete, or misleading, or even completely wrong, which makes debugging the failure more difficult. You can always provide extra `msgAndArgs` to `assert.Equal()` as the 4+ argument.
* `assert.Equal()` is less verbose, so it keeps test smaller and easier to read
* `assert.Equal()` reads in the affirmative, where as conditions for `if x != y ...` are always the inverse. I **want** `x == y`, but I have to write `x != y`.

These points may not be relevant in all cases, but it's easier to be consistent and just use it everywhere.